tests/providers/aws/services/ec2/ec2_securitygroup_allow_ingress_from_internet_to_any_port/ec2_securitygroup_allow_ingress_from_internet_to_any_port_test.py

```diff
@@ -240,6 +464,56 @@ def test_ec2_compliant_default_sg_only_open_to_one_port(self):
                     assert sg.resource_details == default_sg_name
                     assert sg.resource_tags == []
 
+    @mock_aws
+    def test_ec2_compliant_default_sg_only_open_to_one_port(self):
```

Comment:
Is this test name right?

Reply:
> Is this test name right?

Of course not, eagle-eyed! 😄 It comes from the **former** check `ec2_securitygroup_allow_ingress_from_internet_to_any_port` which was renamed to `ec2_securitygroup_allow_ingress_from_internet_to_all_ports` which is compliant if not all ports but only 1 port is open (see [code here](https://github.com/prowler-cloud/prowler/pull/3962/files#diff-a7e83fa8c24ed98f345add4b8110752aae3fcec9359046d72d86b764100868bdR186)).
I corrected it and even caught another one of its kind (`test_ec2_default_sgs_with_all_ports_check`versus `test_ec2_default_sgs_with_any_ports_check`). 🎉